app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="ie.appz.fizzbuzz"
+    android:versionCode="1"
+    android:versionName="1.0" >
+
+    <uses-sdk
```

Comment:
This is no longer needed, this is now handled in the gradle file. Please remove.